packages/domains-table/src/domains-table/domains-table.tsx

```diff
@@ -190,8 +191,10 @@ export const DomainsTable = ( props: DomainsTableProps ) => {
 			return selectedDomainsCopy;
 		} );
 	}, [ domains ] );
-
-	const domainsTableColumns = isAllSitesView ? allSitesViewColumns : siteSpecificViewColumns;
+	const translate = useTranslate();
```

Comment:
wish I didn't have to do this but it's needed by the `resolveDomainStatus` function used in sorting

Reply:
I think we need this anyway because we are sorting based on the translated text.